app/views/katello/api/v2/alternate_content_sources/permissions.rabl

```diff
@@ -0,0 +1,8 @@
+collection @alternate_content_sources
+
+if params[:include_permissions]
```

Comment:
The params here might be strings, which will always be truthy.

I curled with `/katello/api/v2/alternate_content_sources?include_permissions=false"`